docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os.path
+from os import path
+from pathlib import Path
+
+home = str(Path.home())
+exercise_commands = ['mkdir', 'tree']
+
+def get_student_commands():
```

Comment:
thanks @AnnaDinaburgVulikh 
one more thing, if we want to use these functions in other exercises, I think it's better to extract all these functions to a helper file: 
* get_student_command
* used_right_command
*created_directories